svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java

```diff
@@ -0,0 +1,22 @@
+package gov.va.vro.services.bie.config;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import java.util.Map;
+
+@Component
+@ConfigurationProperties(prefix = "bie")
+@Setter
+@Getter
+public class BieProperties {
+
+  /**
```

Comment:
I would give this variable a longer more descriptive name like kafkaTopicMapRMQque.